ui/src/components/pipeline/graph/Node.css

```diff
@@ -17,4 +17,24 @@
 
 .node-tooltip {
     font-size: 1.2em;
+}
+
+.node-watermark {
+    width: 50px;
```

Comment:
fit-content maybe better than fixed width. Not sure if a high TPS would make it overflow or not.